docs/whatToTellStudents.md

```diff
@@ -10,41 +10,41 @@ See the [Assessment page](assessment/index.md) for a description of how grading
 
 HW1.1 is due at 10:59AM on Friday, January 27. There is no need to "submit" your homework. The system will record whatever your score is at that time.
 
 However, you'll note the following line at the top of your screen:
 
-<blockquote>
-Available credit: 110% until 10:59, Wed, Jan 25
-</blockquote>
+> Available credit: 110% until 10:59, Wed, Jan 25
 
-What this means is that if you reach 100% prior to 10:59AM on <em>Wednesday</em> - i.e., complete the homework early - you will receive an extra 10% bonus. You will see this reflected in your score (the instant you reach 100%, it will jump to 110%).
+What this means is that if you reach 100% prior to 10:59AM on _Wednesday_ - i.e., complete the homework early - you will receive an extra 10% bonus. You will see this reflected in your score (the instant you reach 100%, it will jump to 110%).
 
 If you click on the "?" just to the right of the line about available credit, you'll see all the dates associated with this homework. In particular, it says:
 
-<ul><li>you can receive 110% until 10:59AM, Wed, Jan 25</li><li>you can receive 100% until 10:59AM, Fri, Jan 27</li><li>you can receive 80% until 10:59AM, Fri, Feb 10 (two weeks later)</li><li>you can receive 0%, but can keep doing problems as practice for the rest of the semester.</li></ul>
+- you can receive 110% until 10:59AM, Wed, Jan 25
+- you can receive 100% until 10:59AM, Fri, Jan 27
+- you can receive 80% until 10:59AM, Fri, Feb 10 (two weeks later)
+- you can receive 0%, but can keep doing problems as practice for the rest of the semester.
 
-Note that your score will <em>never</em> go down. For example, if you achieve 90% by 10:59AM on Friday, January 25, you won't be able to increase your score after that time, but you won't be penalized for not reaching 100% - your score will remain 90% forever. On the other hand, if you achieve only 70% by 10:59AM on Friday, January 25, you <em>will</em> be able to increase your score after that time (to a maximum of 80%).
+Note that your score will _never_ go down. For example, if you achieve 90% by 10:59AM on Friday, January 25, you won't be able to increase your score after that time, but you won't be penalized for not reaching 100% - your score will remain 90% forever. On the other hand, if you achieve only 70% by 10:59AM on Friday, January 25, you _will_ be able to increase your score after that time (to a maximum of 80%).
 
 I hope this will all make sense once you've actually tried the homework, but please let me know if you have any questions.
 
 If you believe there is a typo or an error in a question, or if you believe your answer was graded incorrectly, please take a screenshot and let me know with a private piazza post. I have access to all of your submissions and can easily check to see what, if anything, went wrong.
-
```

Comment:
Why was this newline removed?

Reply:
This was from the markdown formatter.

Reply:
This doesn't make sense to me, other paragraphs still have newlines between them. I can't see what's different about this one. And when I restore the newline locally and run `make lint-markdown`, it doesn't complain.

Reply:
Sorry, to be clear, this is a VSCode markdown formatter thing on my side.
![image](https://github.com/user-attachments/assets/ca2d8ee5-56b8-412a-a886-b89202ce91b5)
![image](https://github.com/user-attachments/assets/d62437d0-c896-427a-ad8b-df5344c2ea31)


Would you mind sharing your VSCode configuration? Somehow mine is messed up.

![image](https://github.com/user-attachments/assets/6ffa422e-732c-49aa-bcff-ac671774e8e5)

Reply:
> what's different about this one

Does this one have two newlines, is that why?

Reply:
My concern is not with double newlines; Prettier is correctly removing those. My concern is with the newline before "One last note..." being removed.

Reply:
I'd recommend viewing the diff on the "Files" tab on GitHub to make it clear what I'm talking about:

<img width="1279" alt="Screenshot 2025-02-10 at 17 21 45" src="https://github.com/user-attachments/assets/2691c5cd-d321-4b04-a741-dfa8cf0b179c" />

Reply:
Oops🤦‍♂️ Idk how I missed that line